test/test_strcasecmp.cpp

```diff
@@ -0,0 +1,58 @@
+// Copyright 2020 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <gtest/gtest.h>
+
+#include "rcutils/strcasecmp.h"
+
+// Tests the rcutils_strcasecmp() function.
+TEST(TestStrcasecmp, test_strcasecmp) {
+  EXPECT_EQ(0, rcutils_strcasecmp("", ""));
```

Comment:
Do we need to add some test cases if arguments are NULL.

Reply:
will do